css/style.css

```diff
@@ -250,6 +332,10 @@ small {
   .footer-list-links li:last-child {
     border: none;
```

Comment:
You can use bootstrap class for this.

Reply:
Thanks, I didn't know how but now I learned something new :)